chevah_build

```diff
@@ -661,7 +707,7 @@ command_test() {
             # available, but they need to be requested for the tests to pass.
             export LC_ALL=en_US.UTF-8
             execute $LOCAL_PYTHON_BINARY \
-                ../python-modules/python-scandir-1.5/test/run_tests.py
+                ../python-modules/python-scandir-1.7/test/run_tests.py
```

Comment:
maybe we can store the scandir version in a varialble

Reply:
OK.

Reply:
Have actually generalized the idea, as it makes it easier to see where we use custom libs. So:
```
# Python modules versions to be used everywhere possible.
CFFI_VERSION="1.11.5"
PYCRYPTO_VERSION="2.6.1"
PYOPENSSL_VERSION="17.0.0"
CRYPTOGRAPHY_VERSION="2.2.2"
SETPROCTITLE_VERSION="1.1.10"
SCANDIR_VERSION="1.7"
```
And then, where's a change you see something along the lines of:
```
# cryptography 1.3.4 is latest version with support for OpenSSL 0.9.8.
PIP_LIBRARIES_OPENSSL_098="\
    setproctitle==${SETPROCTITLE_VERSION} \
    cryptography==1.3.4 \
    pyOpenSSL==16.0.0 \
    scandir==${SCANDIR_VERSION} \
    "
```